client/elements/sc-navdrawer.html

```diff
@@ -304,6 +305,11 @@ <h2 class="menu-header">Publication details</h2>
           _shouldShowLoadingIndicator(loading, error) {
               return !!(loading && !error);
           }
+
+          _getApiUrl() {
+            let getUrl = window.location;
```

Comment:
`getUrl` feels weird for a variable name. I'd name it either `currentUrl` or `location`